makeMatls.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+#
+from pyne.material import Material,MaterialLibrary
+print "Welcome!"
+mat_lib=MaterialLibrary()
+#
+# define iron for a shield
+ironvec={260000:1} # pure iron
+iron=Material()
+iron.density=7.86
+iron.from_atom_frac(ironvec)
+iron=iron.expand_elements()
+# to write out an mcnp material card
+iron.metadata['mat_number']=101
+iron.write_mcnp('twoslab_mcard.txt', 'atom')
```

Comment:
I think we can write this in one step now from the `MaterialLibrary`

Reply:
I know you can do this for a pyne Material but I didn't see this in the pyne documentation for a MaterialLibrary.   By the way, I assume you mean some function to write out the entire material library in MCNP mcard format to a file.